qdb/qdb.go

```diff
@@ -7,6 +7,35 @@ import (
 	"github.com/pg-sharding/spqr/pkg/config"
 )
 
+type ShardingSchemaKeeper interface {
+	/* persist start of key range move in distributed storage */
+	RecordKeyRangeMove(ctx context.Context, m *MoveKeyRange) error
+	/* list all key-range moves in progress */
+	ListKeyRangeMoves(ctx context.Context) ([]*MoveKeyRange, error)
+	/* mark key range move as completed */
+	UpdateKeyRangeMoveStatus(ctx context.Context, moveId string, s MoveKeyRangeStatus) error
```

Comment:
There's no need for comments here as I can read the method names myself, and they are quite obvious.

Reply:
These is never no need for comments